src/Nethermind/Nethermind.TxPool/TxPool.cs

```diff
@@ -519,14 +514,11 @@ private AcceptTxResult AddCore(Transaction tx, TxFilteringState state, bool isPe
 
         private void UpdateBuckets()
         {
-            lock (_locker)
```

Comment:
Need @marcindsobczak to double check if lock is required or not.

Reply:
My understanding is the blob and transactions pools are mutually exclusive independent pools. A txn goes in one or other but not both, determined by the txn and not the pools; so they don't need to be bonded in this way.

Is very high contention on gnosis testnet over this lock; and it breaks block production as both their is a high level of spam txn and block production is coming from the txn pool rather than block builders.

However I do see intermittent failures with `ProcessedTransactionsDbCleanerTests` tests on decoding

Reply:
Don't see how related though as don't touch locks or tx pool, or are even multithreaded